packages/nc-gui/composables/useLTARStore.ts

```
@@ -201,9 +201,9 @@ const [useProvideLTARStore, useLTARStore] = useInjectionState(
           relatedTableDisplayValueProp.value,
           ...relatedTablePrimaryKeyProps.value,
           attachmentCol.value?.title,
-          ...(fields.value || [])?.map((f) => f.title as string),
+          ...(fields.value || [])?.map((f) => f.title?.trim() as string),
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)